app/workers/metrics/record_daily_usage_measurables_worker.rb

```diff
@@ -0,0 +1,17 @@
+module Metrics
+  class RecordDailyUsageMeasurablesWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform
+      articles_min_15_score_past_24h = Article.published.where("score >= 15").size
+      DataDogStatsClient.count("articles.min_15_score_past_24h", articles_min_15_score_past_24h, tags: { resource: "articles" })
+
+      first_articles_past_24h = Article.where(nth_published_by_author: 1).size
```

Comment:
is there a way to attach descriptions to these when sent to DD? the name `first_articles_past_24h` is not exactly self explanatory. What are we tracking here? What is `nth_published_by_author`?

Reply:
I'm not sure about the first question, @mstruve?

`nth_published_by_author` is in relation to the number of articles the user has published. My first published article is `nth_published_by_author: 1`, my second is `nth_published_by_author: 2`, etc.

Reply:
Oooh! Got it, I would have used a window function to rank articles :D

Reply:
Best way to attach additional data to Datadog metrics is with tags. Then you can display a graph for a metric but split it up by tags or average by tags or do one metric per tags. For example for the postgres db tables I did `postgres.db_table_size` and then added a tag for each table name. Then when I collect the metric I have it split it by that tag.

Reply:
![Screen Shot 2020-01-16 at 12 33 15 PM](https://user-images.githubusercontent.com/1813380/72548306-67edb600-385c-11ea-82a1-1da6ac730a10.png)